github/downloader_test.go

```diff
@@ -1,13 +1,17 @@
 package github
 
 import (
+	"bytes"
 	"context"
+	"encoding/gob"
 	"encoding/json"
 	"fmt"
 	"io/ioutil"
+	"net/http"
 	"os"
 	"testing"
 
+	"github.com/shurcooL/githubv4"
 	"github.com/src-d/metadata-retrieval/testutils"
```

Comment:
nitpick: `github.com/src-d/metadata-retrieval/testutils` is internal import, should be located between stds and third-parties